frontend/public/components/cluster-settings/cluster-settings.tsx

```diff
@@ -930,15 +935,21 @@ export const ClusterNotUpgradeableAlert: React.FC<ClusterNotUpgradeableAlertProp
           <Flex>
             {notUpgradeableClusterOperatorsPresent && (
               <FlexItem>
-                <ClusterOperatorsLink queryString="?rowFilter-cluster-operator-status=Cannot+update">
+                <ClusterOperatorsLink
+                  cancel={cancel}
+                  queryString="?rowFilter-cluster-operator-status=Cannot+update"
```

Comment:
I realize this logic predates this pull request, but what is this `Cannot update` filtering about?  I'd expect to filter on `Upgradeable=False`, but possibly something console-side is converting `Upgradeable=False` into `Cannot update`?

Reply:
Exactly.  See https://github.com/openshift/console/blob/29377adf346a2c78c24bddd99b838726d7112a7b/frontend/public/module/k8s/cluster-operator.ts#L15-L18